### DISCUSSION TITLE
Need re-install geometoric after changing the torch version?

### DISCUSSION THREAD

**Discussion by LukeLIN-web** [pyg-team/pytorch_geometric#5914] [Q&A] [answered]
Currently  I have py3.7, cuda10.2,  torch 1.9.0,torch-geometric        2.0.2
I want to upgrade to torch 1.10.0. I wonder whether I need 
`pip uninstall torch-scatter torch-sparse torch-geometric`
and reinstall all. 

I fail from following command:
```
conda remove pytorch
pip install torch-1.10.0+cu102-cp37-cp37m-linux_x86_64.whl
```
I obtain : OSError: /opt/conda/lib/python3.7/site-packages/torch_sparse/_convert_cuda.so: undefined symbol: _ZNK2at6Tensor5zero_Ev 
I refer https://github.com/pyg-team/pytorch_geometric/issues/1080 , told me that I need pip install from whl. Now I dont know what I am wrong.

**Comment by rusty1s** [comment, marked as answer]
Yes, all extension packages need to be re-installed when updating PyTorch :(
